Cargo.lock

```diff
@@ -367,7 +367,7 @@ dependencies = [
 
 [[package]]
 name = "glean-build"
-version = "6.3.0"
+version = "6.4.0"
```

Comment:
huh, was this missed somewhere? Interesting.

Reply:
Probably from the 5 times I rebased on commits that landed yesterday 😭

Reply:
yeah, no, that was me. I'll fix this next week and make sure upgrading glean-parser correctly changes this!